### ISSUE TITLE
Possible issue regarding the (unsupported) Vive fork and prebuilt executables

### ISSUE THREAD

**Issue by shidarin** [mabrowning/minecrift#172]
Hi guys. I don't know if you care about this or not.

With the release of the vive with it's motion controllers, interest in Minecrift with Vive has apparently exploded, with most people using the [https://github.com/Automat-GH/minecrift](Automat-GH) fork, which I know @mabrowning has expressed concern over the amount of changes to that code.

Unfortunately, the manual build process is apparently too much for some users, and we've had [different](https://www.reddit.com/r/Vive/comments/4fbupb/how_to_set_up_a_vive_minecraft_server/) [prebuilt](https://www.reddit.com/r/Vive/comments/4fbwyb/minecraft_vive_forge_ftb_install_kit_guide_inside/) [executables](https://www.reddit.com/r/Vive/comments/4f5upo/minecraft_for_vive_in_vr_is_our_first_aaa_title/) by random people popping up almost every day in [/r/vive](http://www.reddit.com/r/vive). Today [Kotaku](http://kotaku.com/minecraft-in-virtual-reality-is-the-real-deal-1771828289) linked to one. The people downloading these have no idea that it's multiple degrees away from the original minecrift project, and think they're downloading something official from the minecrift team.

Maybe you don't care about this, but I thought that you'd want a heads up. You obviously can't stop them (Minecrift is under LGPL obviously), but it would help to have an official source for the vive mod.

@Automat-GH has apparently disappeared, with no contact info on his profile, and this being the only repository. 

Lastly, thanks for Minecrift. I built the Automat fork myself and Minecraft in VR is amazing.

**Comment by StellaArtois** [comment]
At some point I'd look to pull in the fork changes back into our repo, but I haven't looked at the code changes yet. 
If @mabrowning reckons the fork code has been hacked to death with a lot of changes, then pulling it in may be somewhat tricky - and maybe not worth the effort to merge. However we may selectively port in the pertinent feature changes while keeping our current code structure.
Not sure what any timeframe for this might be however.

**Comment by Automat-GH** [comment]
Yes, I wouldn't recommend porting my changes wholesale back to the main branch. I made a number of changes that are good for the Vive but not for other control schemes.

**Comment by kylebeee** [comment]
Would it be possible to bind traditional movement to the pointing direction of the left controller as an alternative to teleportation? I'm curious how difficult this would be to implement. Lots of people have been asking about it

**Comment by Automat-GH** [comment]
You can switch to traditional movement by pressing right control + R. It moves in the direction you're facing though.
